doc/api/sending-moments-events.md

```diff
@@ -1,13 +1,67 @@
 # SteelSeries Moments Event Support #
 
-Moments is a new service from SteelSeries GG that allows users to capture gameplay clips, edit them, and share them with the community. When Moments game capture is enabled users can manually clip the game they're playing via a shortcut. Devs can utilize timeline events to mark relevant game events on the clip editing timeline. This makes it easy for users to find the exact moment cool things happen in your game, so they can more easily edit and share the clip.
+Moments is a new service from SteelSeries GG that allows users to capture gameplay clips, edit them, and share them with the community. When Moments game capture is enabled users can manually clip the game they're playing via a shortcut, or enable autoclipping to automatically capture their best gameplay moments. For Autoclipping, devs provide the autoclip conditions and the user can enable or disable them in Moments settings. For Timeline events, devs can utilize timeline events to mark relevant game events on the clip editing timeline. This makes it easy for users to find the exact moment cool things happen in your game, so they can more easily edit and share the clip.
 
 **This feature requires SteelSeries GG which can be downloaded [here](https://steelseries.com/gg/moments)**
 ![timeline events](/images/timelineevents/timeline_events.png)
 
 
 Moments APIs utilize the same server as GameSense events, so adding support for Moments in your existing GameSense game is very simple. You can learn more about server discovery and GameSense events [here](sending-game-events)
 
+
+## Autoclipping ##
+
+Autoclipping works in two parts, registration and triggering. First you need to register a list of events you want to Autoclip. This should be sent via HTTP POST when your game starts to the URL `http://127.0.0.1:<port>/register_autoclip_rules`. It takes in a JSON list of autoclip rules and keys outlined below.
+
+The next step is triggering the autoclip when something cool happens. To do this, HTTP POST the autoclip rule key to the URL `http://127.0.0.1:<port>/autoclip` when your autoclip condition is met in-game.
+
+_autoclip_registration_
+```
+`game`: <string>                        game name that matches what is used for sending gamesense events    mandatory
+`rules`: <autoclip-rules-definition>    list of autoclip rules with keys                                    mandatory
+```
+
+_autoclip-rules-definition_
+```
+`label`: <string>                 non-localized label   optional
+`localized_label_key`: <string>   localization key      optional*
+`default_enabled`: <int> (0-1)    enabled by default    optional - default `0`
```

Comment:
This should be a boolean instead of an integer flag if the only values are 0 and 1.